package.json

```diff
@@ -27,7 +27,9 @@
     "lint": "npx tsc --noEmit && prettier . --check && npm run lint:eslint && npm run lint:markdown",
     "lint:fix": "prettier . --write && npm run lint:eslint --fix && npm run lint:markdown --fix",
     "pre-build": "npx tsx ./scripts/pre-build.ts",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "heroku-cleanup": "npx patch-package",
```

Comment:
For other reviewers, this is necessary to ensure the patches get applied correctly in the Heroku environment, which installs dependencies a bit differently.